crates/next-api/src/dynamic_imports.rs

```diff
@@ -103,6 +103,23 @@ pub(crate) async fn collect_evaluated_chunk_group(
     .await
 }
 
+#[turbo_tasks::value(shared)]
+pub struct NextDynamicImportsResult {
+    pub client_dynamic_imports: IndexMap<Vc<Box<dyn Module>>, DynamicImportedModules>,
+    pub visited_modules: Vc<VisitedDynamicImportModules>,
+}
+
+#[turbo_tasks::value(shared)]
```

Comment:
```suggestion
#[turbo_tasks::value(transparent)]
```

Reply:
Maybe don't make this type public

Reply:
The reason I added this type is to that
- this is a returned value from `collect_next_dynamic_imports`, so it has to be public
- the member is private, so this acts as a opaque wrapper to hand out to the caller